src/Toolkit/Toolkit.UI.Controls/ScaleLine/ScaleLine.Windows.cs

```diff
@@ -141,6 +160,65 @@ private void SetVisibility(bool isVisible)
         {
             Visibility = isVisible ? Visibility.Visible : Visibility.Collapsed;
         }
+#if WPF || WINDOWS_XAML
+        /// <inheritdoc />
+        public partial class ScaleLineAutomationPeer : FrameworkElementAutomationPeer, IValueProvider
+        {
+            private readonly ScaleLine _owner;
+
+            internal ScaleLineAutomationPeer(ScaleLine owner) : base(owner)
+            {
+                _owner = owner;
+            }
+            /// <inheritdoc />
+            protected override string GetLocalizedControlTypeCore() => Properties.Resources.GetString("ScaleLineAutomationTypeName")!;
+            /// <inheritdoc />
+            protected override AutomationControlType GetAutomationControlTypeCore()
+            {
+                return AutomationControlType.Text;
+            }
```

Comment:
Nitpick: Before and after this you use the short-hand `=>` syntax, but you don't here. For consistency consider also doing that here. 

```suggestion
            protected override AutomationControlType GetAutomationControlTypeCore() => AutomationControlType.Text;
```